tests/run-pass/uninit_integer_no_flag.rs

```diff
@@ -0,0 +1,6 @@
+// This test is from https://github.com/rust-lang/miri/issues/1340#issue-600900312.
+// This test passes because -Zmiri-check-number-initialization is not passed.
+
+fn main() {
+    let _val = unsafe { std::mem::MaybeUninit::<usize>::uninit().assume_init() };
```

Comment:
Please merge the run-pass tests into one. (There is a non-negligible constant cost per test.)

Reply:
Also judging from `threadleak_ignored`, maybe a better name for the test would be `uninit_integer_ignored`.

Reply:
> Also judging from `threadleak_ignored`, maybe a better name for the test would be `uninit_integer_ignored` or `uninit_number_ignored`.

Although `threadleak_ignored` is a test of `-Zmiri-ignore-leaks`, not a test of no flag passed. But `ignored` might be more descriptive anyway.